modules/cnf-topology-aware-lifecycle-operator-preparing-for-updates.adoc

```diff
@@ -6,51 +6,118 @@
-====
-The `imageContentSources` section, which can be retrieved with the `oc adm release mirror` command, needs to be saved for platform updates.
-For example, this is the output of the `imageContentSources` which contains the information of your mirrors:
+[source,terminal]
+----
+$ oc adm release mirror
+----
 +
+.Example output
+
 [source,terminal]
 ----
 imageContentSources:
-       - mirrors:
-         - registry.example.com:5000/openshift4
-         source: quay.io/openshift-release-dev/ocp-v4.0-art
+ - mirrors:
+   - mirror-ocp-registry.ibmcloud.io.cpak:5000/openshift-release-dev/openshift4
+   source: quay.io/openshift-release-dev/ocp-release
+ - mirrors:
+   - mirror-ocp-registry.ibmcloud.io.cpak:5000/openshift-release-dev/openshift4
+   source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
 ----
++
+[NOTE]
+====
+Save the image signature of the desired platform image that was mirrored. You must add the image signature to the `PolicyGenTemplate` CR for platform updates.
 ====
 
+.. To get the image signature, first, specify the desired {product-title} tag by running the following command:
++
+[source,terminal]
+----
+$ OCP_RELEASE_NUMBER=<release_version>
+----
+
+.. Specify the architecture of the server by running the following command:
++
+[source,terminal]
+----
+$ ARCHITECTURE=<server_architecture>
+----
+
+.. Get the release image digest from Quay by running the following command
++
+[source,terminal]
+----
+$ DIGEST="$(oc adm release info quay.io/openshift-release-dev/ocp-release:${OCP_RELEASE_NUMBER}-${ARCHITECTURE} | sed -n 's/Pull From: .*@//p')"
+----
+
+.. Set the digest algorithm by running the following command:
++
+[source,terminal]
+----
+$ DIGEST_ALGO="${DIGEST%%:*}"
+----
+
+.. Set the digest signature by running the following command:
++
+[source,terminal]
+----
+$ DIGEST_ENCODED="${DIGEST#*:}"
+----
+
+.. Get the image signature from the link:https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release/[mirror.openshift.com] website by running the following command:
++
+[source,terminal]
+----
+$ SIGNATURE_BASE64=$(curl -s "https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release/${DIGEST_ALGO}=${DIGEST_ENCODED}/signature-1" | base64 -w0 && echo)
+----
+
+.. Save the image signature to the `checksum-<OCP_RELEASE_NUMBER>.yaml` file by running the following commands:
++
+[source,terminal]
+----
+$ cat >checksum-${OCP_RELEASE_NUMBER}.yaml <<EOF
+----
++
+[source,terminal]
+----
+$ {DIGEST_ALGO}-${DIGEST_ENCODED}: ${SIGNATURE_BASE64}
+----
+
 . Prepare the update graph. You have two options to prepare the update graph:
 
-* Use the OpenShift Update Service.
+.. Use the OpenShift Update Service.
 +
 For more information about how to set up the graph on the hub cluster, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.4/html/clusters/managing-your-clusters#build-the-graph-data-init-container[Build the graph data init container].
```

Comment:
We need to use both of the following references to set up upgrade graph using OpenShift Update Service: 

https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.4/html/clusters/managing-your-clusters#deploy-the-operator-for-cincinnati

and 

https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.4/html/clusters/managing-your-clusters#build-the-graph-data-init-container